bin/stencil-download.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+require('colors');
+const apiHost = 'https://api.bigcommerce.com';
+const dotStencilFilePath = './.stencil';
+const options = { dotStencilFilePath };
+const pkg = require('../package.json');
+const Program = require('commander');
+const stencilDownload = require('../lib/stencil-download');
+const versionCheck = require('../lib/version-check');
+const themeApiClient = require('../lib/theme-api-client');
+const inquirer = require('inquirer');
+
+Program
+    .version(pkg.version)
+    .option('--host [hostname]', 'specify the api host', apiHost)
```

Comment:
Can you share some details around how this hostname will be used ?

Reply:
@junedkazi - hostname is all used within https://github.com/bigcommerce/stencil-cli/blob/master/lib/theme-api-client.js, and available within most of the command